Thu Sep 29 03:20:36 MDT 2005  Remko Troncon <remko@psi-im.org>
  * Removed bad includehints.
diff -rN -u old-psi/src/options/opt_appearance-ui.ui new-psi/src/options/opt_appearance-ui.ui
--- old-psi/src/options/opt_appearance-ui.ui	2005-10-31 04:27:31.000000000 -0700
+++ new-psi/src/options/opt_appearance-ui.ui	2005-10-31 04:27:45.000000000 -0700
@@ -519,10 +519,4 @@
     <tabstop>pb_fChat</tabstop>
 </tabstops>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>fontlabel.h</includehint>
-    <includehint>fontlabel.h</includehint>
-    <includehint>fontlabel.h</includehint>
-    <includehint>fontlabel.h</includehint>
-</includehints>
 </UI>
diff -rN -u old-psi/src/ui_about.ui new-psi/src/ui_about.ui
--- old-psi/src/ui_about.ui	2005-10-31 04:27:30.000000000 -0700
+++ new-psi/src/ui_about.ui	2005-09-29 01:25:16.000000000 -0600
@@ -303,7 +303,6 @@
 <layoutdefaults spacing="6" margin="11"/>
 <includehints>
     <includehint>fancylabel.h</includehint>
-    <includehint>urllabel.h</includehint>
     <includehint>psitextview.h</includehint>
     <includehint>psitextview.h</includehint>
     <includehint>psitextview.h</includehint>

